src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Validators/Apprentice/StartDateViewModelValidator.cs

```diff
@@ -34,6 +39,10 @@ public StartDateViewModelValidator()
                 .Must((y, _) => y.StartDate.Date < (new MonthYearModel(y.EndDate).Date))
                 .WithMessage("Enter a start date prior to the new training end date")
                 .When(a => a.EndDate != null && a.StartDate.HasValue && a.StartDate.IsValid);
+            RuleFor(x => x.StartDate)
```

Comment:
Just a personal opinion, can we please change the code to something like , as the academicYear already has a CurrentAcademicYearEndDate property.

startDate <= _academicYear.CurrentAcademicYearEndDate.AddYears(1)

 your code is correct, but to keep it aligned on what we have on V1 and to make it a bit easier to read with the user story i.e. ( I have entered a Month/ Year that is LATER THAN the end of current academic year + 1 Year (i.e., July ))

Reply:
I have updated